sass/_general-mixins.scss

```diff
@@ -121,3 +121,13 @@ $black: #000; // for shade lint pass
 @function shade($color, $percentage) {
   @return mix($black, $color, $percentage);
 }
+
+
+//
+//
+@mixin basic-focus {
+  @include box-shadow(inset 0 0 0 0.15rem $focus-color);
+  border-style: solid;
+  outline: none;
+  padding: ($general-padding / 2) 0;
```

Comment:
Why don't we use the variable that holds `$general-padding / 2` directly but use an operation here? Is that because `$general-padding` changes from different layouts?

Reply:
There is no real good reason why there isnt a specific variable that represents this calculation. However I do know that this kind of pattern is used throughout the DTK. To go through and change all of them to satisfy the issue you raise is beyond the scope of this PR. Please feel free to create an issue, so it can officially be logged.

Reply:
Yeah I agree we shouldn't be working on other cases that use calculation in this PR.

However, I didn't find any `($general-padding / 2)` in `development` branch, so it will be only introduced from this PR. Giving that, focusing on `($general-padding / 2)` in this PR should be inside the scope.

I feel it will be simpler to have `$general-padding / 2` as a new variable for focus padding. So if focus padding needs to be changed in the future, you don't have to consider `$general-padding` at all. I will leave you to do the final decision :-)

Reply:
If you look at the DTK as a whole there are many calculations _of this type_ where we do math against a $variable's value. That is what I was referring to, as out of scope